### ISSUE TITLE
How to connect to Kafka in Confluent

### ISSUE THREAD

**Issue by thanhlmm** [airbytehq/airbyte#8220]
### Tell us about the documentation you'd like us to add or update.
Hi, I'd like to have my Kafka as destination. My kafka is running on Confluent but there is no way I can make it running

```java
# Required connection configs for Kafka producer, consumer, and admin
bootstrap.servers=pkc-ldvr1.asia-southeast1.gcp.confluent.cloud:9092
security.protocol=SASL_SSL
sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule   required username='{{ CLUSTER_API_KEY }}'   password='{{ CLUSTER_API_SECRET }}';
sasl.mechanism=PLAIN
# Required for correctness in Apache Kafka clients prior to 2.6
client.dns.lookup=use_all_dns_ips

# Best practice for Kafka producer to prevent data loss
acks=all

# Required connection configs for Confluent Cloud Schema Registry
schema.registry.url=https://{{ SR_ENDPOINT }}
basic.auth.credentials.source=USER_INFO
basic.auth.user.info={{ SR_API_KEY }}:{{ SR_API_SECRET }}
```

This is a Confluent config for connection. I also found that If I select `SASL_SSL` as Security Protocal, there is no way I can select `sasl.mechanism` as PLAIN
So I don't know if it is a bug or missing concept

### If applicable, add links to the relevant docs that should be updated
- [[Link to relevant docs page](https://docs.airbyte.io)](https://docs.airbyte.io/integrations/destinations/kafka)

**Comment by sumitm-iiit** [comment]
This issue should be fixed now with the fix for  https://github.com/airbytehq/airbyte/issues/11934.
Can we close this ticket ?

cc: @thanhlmm - tagging you if you were waiting for the fix for this issue.

**Comment by mohitreddy1996** [comment]
Hi, 

I am trying to setup Kafka destination connector for a Kafka cluster running on Confluent and need this support.

I tried deploying Airbyte on Kubernetes using (as documented in - https://docs.airbyte.com/deploying-airbyte/on-kubernetes/#launch-airbyte):

```
kubectl apply -k kube/overlays/stable
```

I don't see the option of setting PLAIN in the UI. Attached screenshot - 

![image](https://user-images.githubusercontent.com/11742913/168620642-2f654ebf-05e7-47b4-96f7-50daa4ebaf40.png)

Could there potentially be a versioning issue? (I see that stable version has tags > 0.38. The fix was released in 0.36.X)?

Thanks!
